applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -158,14 +158,88 @@ config BLE_ISO_TEST_PATTERN
 	bool "Transmit a test pattern to measure link performance"
 	default n
 	help
-	 This will transmit the same amount of data as an audio stream, but the
-	 data will be an incrementing value ranging from 0-255 and repeating.
-	 Note that enabling this feature will disable the audio stream.
+	  This will transmit the same amount of data as an audio stream, but the
+	  data will be an incrementing value ranging from 0-255 and repeating.
+	  Note that enabling this feature will disable the audio stream.
 
 config BLE_ISO_RX_STATS_S
 	int "Interval in seconds to print BLE ISO RX stats. 0 to deactivate"
 	default 0
 
+choice BT_AUDIO_UNICAST_BAP_CONFIGURATION
+	prompt "Unicast codec configuration"
+	depends on TRANSPORT_CIS
+	default BT_AUDIO_UNICAST_RECOMMENDED
+	help
+	  Select the unicast codec configuration as given in
+	  Table 5.2 of BAP Specification, USB only supports BT_AUDIO_UNICAST_RECOMMENDED
+
+config BT_AUDIO_UNICAST_RECOMMENDED
+	bool "Recommended unicast codec capability"
+	depends on TRANSPORT_CIS
+	help
+	  Recommended unicast settings for the nRF5340_audio application
+	  48kHz, 96kbps, 2 retransmits, 20ms transport latency, and 10ms presentation delay
+
+config BT_AUDIO_UNICAST_16_2_1
+	bool "16_2_1"
+	depends on TRANSPORT_CIS
+	help
+	  Unicast mandatory codec capability 16_2_1
+	  16kHz, 32kbps, 2 retransmits, 10ms transport latency, and 40ms presentation delay
+
+config BT_AUDIO_UNICAST_24_2_1
+	bool "24_2_1"
+	depends on TRANSPORT_CIS
+	help
+	  Unicast mandatory codec capability 24_2_1
```

Comment:
We can remove "mandatory" for 24KHz sampling rate, since it is removed in the latest BAP.